### ISSUE TITLE
Enable the google translate dropdown on all pages

### ISSUE THREAD

**Issue by GideonShils** [Zmwang622/Vaccinate-PA#45]
See the description in this PR: https://github.com/Zmwang622/Vaccinate-PA/pull/43

tl;dr:

The google translate widget works by loading a script on page load, and inserting the translation dropdown on the page where the translation div is located. Since the script only loads once, but the footer reloads on page change, consecutive page changes cause the translation dropdown to disappear.

We can potentially fix this by constructing the script in a way where it reloads on page change to ensure the div gets recreated each change. Need to be careful here that we don't end up accidentally loading multiple copies of the script (might need to tear down previously loaded version?)
